Library/Homebrew/formula.rb

```diff
@@ -690,10 +695,37 @@ def bottle_prefix
     prefix+".bottle"
   end
 
-  # The directory where the formula's installation logs will be written.
+  # The directory where the formula's installation or test logs will be written.
   # @private
-  def logs
-    HOMEBREW_LOGS+name
+  def logs(log_type = :main)
+    case log_type
+    when :main
```

Comment:
And I think it should be avoid, you can always pass the type in the sandbox rules, which is the only place it's used.